docs/source/guide/Chains.lrad

```diff
@@ -31,51 +31,55 @@ In other words, we would like for programs to be *deterministic*.
 
 TODO: more.
 
-## Chains
+## Exploring remote chains
 
-### Remote chains
+First we start up a simple server that hosts remote chains on port 8023.
 
-The basic building blocks of interacting with remote chains are the `send!` and
-`receive!` operations. These send expressions to a remote chain, or receive new
-ones since a given index, respectively.
+```
+stack exec -- radicle-server 8023
```

Comment:
This has changed in master in a couple of ways. First, we now need psql. Second, the port is a named arg.